### DISCUSSION TITLE
Parallel mode loading causing freeze in Reaper

### DISCUSSION THREAD

**Discussion by andivax** [apohl79/audiogridder#677] [Help]
Hi guys!
Today I have tested "allow plugins to be loaded in parallel" option and it is definitely has a bug.
I am using Reaper 6 x64 under Win 10 Pro x64

To reproduce:
1) load audiogridder fx vst3 x64
2) insert Acustica Audio plugin (I am using Celestial ME demo) + IK Multimedia Stealth Limiter
3) Save project and re-load it
After this just try to open audiogridder and boom - Reaper freezes forever.

**Comment by apohl79** [comment]
Some plugins cause issues, thats why this option was added. I will remove it at some point and keep the behavior with it being disabled. True parallel loading works only with sandboxing.

**Comment by andivax** [comment]
thank you for the clarification!

what about not worked right click on server and settings buttons in
audiogridder plugin?
did you confirm this behaviour? previous beta 03 was ok

пт, 30 июл. 2021 г. в 21:08, Andreas Pohl ***@***.***>:

> Some plugins cause issues, thats why this option was added. I will remove
> it at some point and keep the behavior with it being disabled. True
> parallel loading works only with sandboxing.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/677#discussioncomment-1092493>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARLSQUPONEBSAD5HXVGN2DLT2LTARANCNFSM5BIPB77Q>
> .
>


-- 

<http://www.andivaxmastering.com/>

Andrii "ANDI VAX" Vakhnenko

www.ANDIVAXMASTERING.com <http://www.andivaxmastering.com/>

[image: Facebook] <https://www.facebook.com/groups/207215009301681/>   [image:
Youtube] <https://www.youtube.com/user/andivax>

**Reply by apohl79** [reply]
Nope, haven't seen this. Works for me.

**Reply by apohl79** [reply]
if the menu doesn't show, is reaper hanging? if yes, can you provide a core dump? https://audiogridder.com/bug-reports/#coredump

**Comment by andivax** [comment]
hmmm. ok
I will test previous build again and then last 04 build on Monday

have a nice weekend!

пт, 30 июл. 2021 г. в 21:18, Andreas Pohl ***@***.***>:

> Nope, haven't seen this. Works for me.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/677#discussioncomment-1092529>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARLSQUIHID4J27K42DPHSBDT2LUJFANCNFSM5BIPB77Q>
> .
>


-- 

<http://www.andivaxmastering.com/>

Andrii "ANDI VAX" Vakhnenko

www.ANDIVAXMASTERING.com <http://www.andivaxmastering.com/>

[image: Facebook] <https://www.facebook.com/groups/207215009301681/>   [image:
Youtube] <https://www.youtube.com/user/andivax>

**Comment by andivax** [comment]
No hangs
Just menu is not appearing

Will send core dump on Monday
In the road already ) sorry

Пт, 30 июля 2021 г. в 21:38, Andreas Pohl ***@***.***>:

> if the menu doesn't show, is reaper hanging? if yes, can you provide a
> core dump? https://audiogridder.com/bug-reports/#coredump
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/677#discussioncomment-1092621>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARLSQUMS7YR72GTDNPDQX4LT2LWRJANCNFSM5BIPB77Q>
> .
>
-- 
www.ANDIVAXMASTERING.com

**Comment by andivax** [comment]
HI!
I have downgraded to beta 3 with NO luck. Right clicking on plugin GUI does
not work.
Upgraded to beta 4 and maked memory dump: https://we.tl/t-Sqq05Gcc5u

пт, 30 июл. 2021 г. в 21:41, Andi Vax ***@***.***>:

> No hangs
> Just menu is not appearing
>
> Will send core dump on Monday
> In the road already ) sorry
>
> Пт, 30 июля 2021 г. в 21:38, Andreas Pohl ***@***.***>:
>
>> if the menu doesn't show, is reaper hanging? if yes, can you provide a
>> core dump? https://audiogridder.com/bug-reports/#coredump
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/apohl79/audiogridder/discussions/677#discussioncomment-1092621>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ARLSQUMS7YR72GTDNPDQX4LT2LWRJANCNFSM5BIPB77Q>
>> .
>>
> --
> www.ANDIVAXMASTERING.com
>


-- 

<http://www.andivaxmastering.com/>

Andrii "ANDI VAX" Vakhnenko

www.ANDIVAXMASTERING.com <http://www.andivaxmastering.com/>

[image: Facebook] <https://www.facebook.com/groups/207215009301681/>   [image:
Youtube] <https://www.youtube.com/user/andivax>

**Comment by andivax** [comment]
Removing all Audiogridder pligin and server preferences is not helping.

Also one more thing - if path to plugin is very long - it won't fit in
string (please fix this): https://prnt.sc/1idmejs

пн, 2 авг. 2021 г. в 16:32, Andi Vax ***@***.***>:

> HI!
> I have downgraded to beta 3 with NO luck. Right clicking on plugin GUI
> does not work.
> Upgraded to beta 4 and maked memory dump: https://we.tl/t-Sqq05Gcc5u
>
> пт, 30 июл. 2021 г. в 21:41, Andi Vax ***@***.***>:
>
>> No hangs
>> Just menu is not appearing
>>
>> Will send core dump on Monday
>> In the road already ) sorry
>>
>> Пт, 30 июля 2021 г. в 21:38, Andreas Pohl ***@***.***>:
>>
>>> if the menu doesn't show, is reaper hanging? if yes, can you provide a
>>> core dump? https://audiogridder.com/bug-reports/#coredump
>>>
>>> —
>>> You are receiving this because you authored the thread.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/apohl79/audiogridder/discussions/677#discussioncomment-1092621>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/ARLSQUMS7YR72GTDNPDQX4LT2LWRJANCNFSM5BIPB77Q>
>>> .
>>>
>> --
>> www.ANDIVAXMASTERING.com
>>
>
>
> --
>
> <http://www.andivaxmastering.com/>
>
> Andrii "ANDI VAX" Vakhnenko
>
> www.ANDIVAXMASTERING.com <http://www.andivaxmastering.com/>
>
> [image: Facebook] <https://www.facebook.com/groups/207215009301681/>   [image:
> Youtube] <https://www.youtube.com/user/andivax>
>


-- 

<http://www.andivaxmastering.com/>

Andrii "ANDI VAX" Vakhnenko

www.ANDIVAXMASTERING.com <http://www.andivaxmastering.com/>

[image: Facebook] <https://www.facebook.com/groups/207215009301681/>   [image:
Youtube] <https://www.youtube.com/user/andivax>
